spec/services/dashboard_event_handler_spec.rb

```diff
@@ -95,6 +95,19 @@
       end
     end
 
+    context "saved with_state abandoned" do
+      let(:state) { "abandoned" }
+
+      it "does not fire additional Application jobs" do
+        # one is fired from creating the LegalAidApplication required by the ccms_submission factory
```

Comment:
**not a blocker** but I am, after reading this, still unclear why this test and comment are similar to `initialised` but not failed/completed states.